### DISCUSSION TITLE
Is lastest version of AspNetCoreOdata compliant with Odata 4.01 ?

### DISCUSSION THREAD

**Discussion by danleydmello** [OData/AspNetCoreOData#570] [General]
We are using latest "Microsoft.AspNetCore.OData" Version="8.0.8" 
In the odata response , it outputs **odata-version: 4.0**  ,  Is this odata version 4.0 ?

![image](https://user-images.githubusercontent.com/16119774/164109371-1539ca48-0eb6-4307-b08a-3a7120a220c4.png)

**Comment by TehWardy** [comment]
What are you thinking is missing / the difference here?

**Reply by danleydmello** [reply]
@TehWardy , sorry didn't understand your response? I just want to know if latest version of AspNetCoreData supports OData 4.01 specs ?

**Comment by TehWardy** [comment]
4.01 is basically the same as 4.0 ... so my question was ... what are you suspecting might be missing?
